pallets/asset-registry/src/lib.rs

```diff
@@ -528,15 +581,29 @@ impl<T: Config> Pallet<T> {
 	pub fn get_or_create_asset(
 		name: Vec<u8>,
 		asset_type: AssetType<T::AssetId>,
-		existential_deposit: T::Balance,
+		existential_deposit: Balance,
 		asset_id: Option<T::AssetId>,
+		is_sufficient: bool,
 	) -> Result<T::AssetId, DispatchError> {
-		let bounded_name: BoundedVec<u8, T::StringLimit> = Self::to_bounded_name(name)?;
+		let bounded_name = Self::try_into_bounded(Some(name))?;
 
-		if let Some(asset_id) = AssetIds::<T>::get(&bounded_name) {
+		//NOTE: this unwrap is safe.
```

Comment:
Pls explain why in comment if you do smth like this

Reply:
should be an `expect`, no?